src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithoutStrongNames.txt

```diff
@@ -171,11 +171,12 @@ Referenced assemblies:
 - 'Microsoft.VisualStudio.Threading, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
 - 'SonarLint.VisualStudio.Core, Version=6.14.0.0, Culture=neutral, PublicKeyToken=null'
+- 'SonarLint.VisualStudio.IssueVisualization, Version=6.14.0.0, Culture=neutral, PublicKeyToken=null'
```

Comment:
The project file hasn't changed in this PR, but the list of referenced assemblies *has* changed.

I'm guessing this is because the previously we we're actually using anything from IssueViz, so the compiled assembly didn't actual need to reference it before.